### PR TITLE
[OpenCL] Cleans dense_update_ops

### PR DESCRIPTION
**Pull request by lukeiwanski**

### CODE REVIEW

File: tensorflow/core/kernels/dense_update_ops.cc [hunk 1]
```diff
@@ -214,4 +207,16 @@ TF_CALL_GPU_NUMBER_TYPES(REGISTER_GPU_KERNELS);
 #undef REGISTER_GPU_KERNELS
 #endif  // end GOOGLE_CUDA
 
+#ifdef TENSORFLOW_USE_SYCL
+#define REGISTER_KERNELS(type)                                         \
```
Comment (gunan) [inline tensorflow/core/kernels/dense_update_ops.cc]: Nit. 
s/REGISTER_KERNELS/REGISTER_SYCL_KERNELS/

Comment (lukeiwanski) [inline tensorflow/core/kernels/dense_update_ops.cc]: Fixed in d53bcfe

**Review by gunan — APPROVED** [review]

**Comment by gunan** [comment]
Jenkins, test this please.

**Comment by gunan** [comment]
Possible infra issue in the failing test.
Jenkins, test this please.

**Comment by gunan** [comment]
known flake, merging.

**Pull request merged**